packages/widget-v2/src/modals/TokenAndChainSelectorModal/TokenAndChainSelectorModal.tsx

```diff
@@ -41,20 +41,18 @@ export type SelectorContext = "source" | "destination";
 export type TokenAndChainSelectorModalProps = ModalProps & {
   onSelect: (token: ClientAsset | null) => void;
   selectedAsset?: ClientAsset;
-  networkSelection?: boolean;
-  context: SelectorContext
+  selectChain?: boolean;
```

Comment:
rename prop to `selectChain`, I think it's best to be consistent and always use the same word when referring to `chains`

Reply:
I'm going to refactor/change the name of this Modal to `AssetsAndChainSelectorModal` for the same reason, but I will do it in a separate PR to not mess with the diff

Reply:
https://github.com/skip-mev/skip-go/pull/388